jbpm-wb-process-runtime/jbpm-wb-process-runtime-api/src/main/java/org/jbpm/workbench/pr/model/DocumentSummary.java

```diff
@@ -24,6 +24,8 @@
 public class DocumentSummary extends GenericSummary<String> {
 
     private String documentId;
+    private String documentServerTemplateId;
+    private String documentIdentifier;
```

Comment:
this documentId is not needed as filename can be taken from DocumentInstance that is returned from getDocument call over DocumentServicesClient.